components/collections/codepointtrie_builder/src/lib.rs

```diff
@@ -71,10 +71,11 @@
 #![cfg_attr(
     not(test),
     deny(
-        clippy::indexing_slicing,
-        clippy::unwrap_used,
-        clippy::expect_used,
-        clippy::panic,
+        // This Datagen-only builder code may panic when interacting with WASM
```

Comment:
nit: datagen is not really a concept in this crate

Reply:
Updated the docs